source/includes/_tipser-elements.md

```diff
@@ -193,3 +193,43 @@ import { ShopComponent } from '@tipser/tipser-elements';
 
 Note: the shop component is updating the top-level page URL (when it's tabs are clicked). For this reason, please double check if it won't interfere with your web framework. 
 For the same reason, it's not recommended to include more than one `<ShopComponent />` on a single page. 
+
+
+## Style Guidelines ##
```

Comment:
Shouldn't the section be called in a way that it indicates, that this is guideline to altering styles by the user ? For me the title does not really say what it is. 

I'm not sure what should be the title tough. "Custom Style Guidelines" / "Custom Styles" / "How to alter default styles" / "Adjusting Style - Guidelines" /  "Customizing styles - Guidelines"

Reply:
I think I like  "Customizing style - Guidelines" the most.

Reply:
I think `Customizing Styles` is enough

Reply:
In react-select library a similar section is called: Customising the styles